daml-service/entrypoint.sh

```diff
@@ -4,6 +4,6 @@ set -euo pipefail
 [[ -n ${VERBOSE:-} ]] && set -x
 
 # Log everything (stdout and stderr) to both log file and terminal
-exec > >(tee /var/log/promtail/http-json.log) 2>&1
+exec > >(tee /var/log/promtail/"${DAML_SERVICE}") 2>&1
 
-eval "exec java -XX:+CrashOnOutOfMemoryError ${JAVA_OPTS:-} -jar /opt/http-json.jar \"\$@\""
+eval "exec java -XX:+CrashOnOutOfMemoryError ${JAVA_OPTS:-} -jar /opt/service.jar \"\$@\""
```

Comment:
We could also use `${DAML_SERVICE}.jar` here and in the `Dockerfile` to avoid confusion in the docker image.

Reply:
When you inspect the image/container content to debug you will have the same file names. I It would be to distinguish the HTTP JSON API from the Trigger I guess.